PKGBUILD

```diff
@@ -25,13 +24,16 @@ depends=('c-ares' 'intel-mkl' 'onednn' 'pybind11' 'openssl' 'lmdb' 'libpng' 'cur
 makedepends=('bazel' 'python-numpy' 'rocm-hip-sdk' 'miopen' 'rccl' 'git'
              'python-pip' 'python-wheel' 'python-setuptools' 'python-h5py'
              'python-keras-applications' 'python-keras-preprocessing'
-             'cython')
+             'cython' 'rocblas' 'rocrand' 'rocfft' 'hipfft' 'roctracer'
+             'hipsparse' 'hipsolver' 'rocsolver' 'rocm-hip-sdk')
 optdepends=('tensorboard: Tensorflow visualization toolkit')
-source=("$pkgname-$pkgver.tar.gz::https://github.com/tensorflow/tensorflow/archive/v${_pkgver}.tar.gz"
-        fix-c++17-compat.patch)
+source=("$pkgname::git+https://github.com/ROCmSoftwarePlatform/tensorflow-upstream.git"
+        fix-c++17-compat.patch
+        hiprand.patch)
```

Comment:
Can you make this patch pull from a pull request instead of manually having the patch in this repo? Having the patch sent upstream helps us verify it via the upstream devs and ensure it gets merged in eventually, where the patch is not even necessary.

Reply:
I need to investigate that further, because i am not shure if the arch packaging is wrong, or if it's a tensorflow problem.